examples/asr/transcribe_speech.py

```diff
@@ -93,6 +94,86 @@ class TranscriptionConfig:
     rnnt_decoding: RNNTDecodingConfig = RNNTDecodingConfig(fused_batch_size=-1)
 
 
+def transcribe_partial_audio(
```

Comment:
These functions are quite large, better to move the into some util dir and import only here.

Reply:
Cannot find a good existing place. How about having a transcribe_utils.py so we can put everything about transcribe there.